### PR TITLE
Use Markdown report for non-inline mode

### PR DESCRIPTION
**Pull request by thii**

### CODE REVIEW

**Comment by thii** [comment]
I disabled testing on Ruby 2.0 on Travis because of an unrelated error.

**Comment by giginet** [comment]
Thank you @thii 

I'm going to review tomorrow. 😪

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ cache:
     - bundle
 
 rvm:
-  - 2.0
```
Comment (giginet) [inline .travis.yml:7]: danger/danger seems not to be supported Ruby 2.0. So, I agree dropping 2.0.

File: lib/xcprofiler/danger_reporter.rb [hunk 2]
```diff
@@ -28,17 +36,38 @@ def report!(executions)
...
-    private
+    def markdown_report(executions)
+      warning_executions = executions.select do |e|
+        e.time >= @thresholds[:warn] && e.time < @thresholds[:fail]
+      end
+      error_executions = executions.select do |e|
+        e.time >= @thresholds[:fail]
+      end
+
+      return unless warning_executions.count > 0 || error_executions.count > 0
```
Comment (giginet) [inline lib/xcprofiler/danger_reporter.rb]: This condition is hard to read.

```ruby
return if warning_executions.empty? && error_executions.empty?
```

File: lib/xcprofiler/danger_reporter.rb [hunk 3]
```diff
@@ -28,17 +36,38 @@ def report!(executions)
...
+      @dangerfile.markdown(message, {})
     end
 
     def relative_path(path)
       working_dir = Pathname.new(@working_dir)
       Pathname.new(path).relative_path_from(working_dir).to_s
     end
+
+    def markdown_issues(executions, heading)
+      message = "#### #{heading}\n\n"
```
Comment (giginet) [inline lib/xcprofiler/danger_reporter.rb:61]: How about using `erb` to render tables.

**Review by giginet — APPROVED** [review]
@thii Thanks for your contribution.
I commented on some points.

**Review by giginet — APPROVED** [review]
LGTM

**Comment by giginet** [comment]
@thii Thank you for your contribution 🎉 
I'll release newer version soon.

**Comment by thii** [comment]
@giginet Thank you and sorry I haven't had time to look at the erb stuff yet (I'm a Ruby newbie ;) )

**Comment by giginet** [comment]
@thii Never mind. You've done a great work 👍

**Comment by giginet** [comment]
@thii I'm sorry. I forgot to release a newer version. 🙇 

I've just released 0.3.0

**Pull request merged**